### PR TITLE
Footer fixes

### PR DESCRIPTION
**Pull request by brett-stephen**
- fixed filters sidebar issues,
- removed ununsed code,
- fixed white link color out of footer

## Briefly describe the feature or bug that this PR addresses.
- general issue clean up from the footer update

### CODE REVIEW

File: src/components/sidebar/right-sidebar/RightSidebar.tsx [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import React from "react";
 import "../sidebar.scss";
-import Datepicker from "./datepicker/Datepicker";
```
Comment (HimanshuRanka) [inline src/components/sidebar/right-sidebar/RightSidebar.tsx:3]: is the datepicker not a part of the right sidebar?

File: src/types.ts [hunk 2]
```diff
@@ -42,7 +42,7 @@ export type AggregatedRecord = {
   seroprevalence: number;
 };
 
-export type Filters = {
+export type FiltersConfig = {
```
Comment (HimanshuRanka) [inline src/types.ts:45]: why the name change?

**Review by HimanshuRanka — APPROVED** [review]
LGTM! just a couple random questions

File: src/types.ts (hunk 2 above)
Comment (brett-stephen) [reply src/types.ts:45]: We already had a `Filters` component, so a `Filters` type was just confusing and hard to work with

File: src/components/sidebar/right-sidebar/RightSidebar.tsx (hunk 1 above)
Comment (brett-stephen) [reply src/components/sidebar/right-sidebar/RightSidebar.tsx:3]: It is, but I believe this was unused code from a different implementation

**Review by HimanshuRanka — APPROVED** [review]

**Pull request merged**